tox.ini

```diff
@@ -34,6 +34,10 @@ commands =
 skip_install = True
 commands = - vulture --exclude="a3m/settings,a3m/main/models.py,a3m/main/migrations,a3m/fpr/models.py,a3m/fpr/migrations,a3m/server/rpc/proto" a3m
 
+[testenv:bandit]
+skip_install = True
+commands = bandit -r a3m --ini .bandit
```

Comment:
Would adding `{toxinidir}` to the `.bandit` path be appropriate here to make it relative to the tox configuration? See the first `covegate` call above.

Reply:
Good idea @replaceafill - Addressed in latest commit.